### PR TITLE
feat(suite): prefill token send form

### LINKED ISSUE #12820: EVM tokens/ Send form adjustment for tokens
after clicking the 'Send' button next to the token, the user should be taken to the send form, where the token would be prefilled

Screenshot 2024-06-11 at 16 59 52

Screenshot 2024-06-11 at 17 00 06

### PR DESCRIPTION
**Pull request by AdamSchinzel**
## Description

When you click send in tokens table it will redirect you to send form with that token prefilled for you.

## Related Issue

Resolve #12820

### CODE REVIEW

File: packages/suite/src/constants/suite/routes.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ type RouteParamsTypes = {
     accountIndex: number;
     accountType: NonNullable<Network['accountType']>;
     cancelable: boolean;
+    contractAddressPrefill?: string;
```
Comment (AdamSchinzel) [inline packages/suite/src/constants/suite/routes.ts]: Was thinking about some better name for this but didn't come up with anything.

Comment (tomasklim) [reply packages/suite/src/constants/suite/routes.ts]: how about just `contractAddress`?

Comment (tomasklim) [reply packages/suite/src/constants/suite/routes.ts]: there is symbol, accountType, i think it is fine to use contractAddress

File: packages/suite/src/utils/suite/router.ts [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const getApp = (url: string) => {
 const validateWalletParams = (url: string): CommonWalletParams => {
     const [, hash] = stripPrefixedURL(url).split('#');
     if (!hash) return;
-    const [symbol, index, type] = hash.split('/').filter(p => p.length > 0);
+    const [symbol, index, contractAddress, type] = hash.split('/').filter(p => p.length > 0);
```
Comment (tomasklim) [inline packages/suite/src/utils/suite/router.ts]: I think that `contractAddress` has to be 4th as it is optional

Comment (tomasklim) [reply packages/suite/src/utils/suite/router.ts]: - `#/btc/0/segwit` 
- `web/accounts/tokens#/eth/1/ledger`

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/utils/suite/router.ts (hunk 2 above)
Comment (AdamSchinzel) [reply packages/suite/src/utils/suite/router.ts]: Yes but there was a problem that `normal` type was excluded on purpose from the hash so I removed this and put the `contractAddress` as the last item.

This should also fix the failing unit tests.

**Review by tomasklim — APPROVED** [review]

**Pull request merged**